conda-recipe/meta.yaml

```diff
@@ -79,12 +79,12 @@ outputs:
         # additional dependencies for convenience in conda-forge
         - fsspec
         - pooch >=1.3.0
-        - zarr
+        - zarr >=2.12.0
 
       run_constrained:
         - pyside2 >=5.14.2,!=5.15.0
-        - pyqt >=5.12.3,!=5.15.0
         - pyside6 <6.3.2
```

Comment:
I see that this constraint is no longer mentioned in pyproject.toml, but [there are several entries in the `denylist`](https://github.com/psobolewskiPhD/napari/blob/2e5a7fe83ec0ad1a626dffbd8255f9f286091ce7/resources/constraints/version_denylist.txt#L3C9-L3C95)... Are we supposed to track those in the conda recipe too? I'm not sure what the role of that constraint file is 🤔 

Do you have more details @psobolewskiPhD?

Reply:
Hmm, I stil see it in pyproject.toml:
https://github.com/napari/napari/blob/c864d6e41a900721517c1cf0654f3b02ec3fcb01/pyproject.toml#L106-L108

Or are you referring to pyside6?
That one is a big problem no combo of napari, python, and pyside6 works 100% properly.
The deny list is used to define what combination can pass on CI, but it has glitches in regular usage including flashing windows and seg faults.
It's very much experimental and not recommended.

Reply:
Yes, I was referring to `pyside6`. Should we prevent its installation altogether then?  I can also update it to:

```
PySide6 <6.5
```

so it tracks `pyside6_experimental`.

Reply:
Right now it look like <6.5 or >= 6.7 might actually result in run-able napari.
With all the other caveats.
I dunno about preventing it all together, maybe people want to test?

Reply:
Ok, let's add `<6.5,>=6.7` then. Thanks!